config/config.go

```diff
@@ -137,7 +137,7 @@ type SmeshingConfig struct {
 	Start           bool                              `mapstructure:"smeshing-start"`
 	CoinbaseAccount string                            `mapstructure:"smeshing-coinbase"`
 	Opts            activation.PostSetupOpts          `mapstructure:"smeshing-opts"`
-	ProvingOpts     activation.PostProvingOpts        `mapstructure:"smeshing-proving-opts"`
+	ProvingOpts     activation.PostProvingOpts        `mapstructure:"smeshing-proving-opts"` // TODO(mafa): remove?
```

Comment:
opinions?

Reply:
IMO this should be kept. Users need to tweak post-proof generation in the supervised post service.

Reply:
Right now these settings are unused (only used in `PostSetupManager::GenerateProof` which is only used in 2 tests). I could pass them to the post-service instead.

I see that it is possible to pass threads and nonces to the post service, but not `flags`; only `randomx-mode`. How do you think `flags` should be handled then?

Reply:
`ProvingOpts` are now passed to the post supervisor when `StartSmeshing == true`